docs/src/content/docs/cookbook/jettons.mdx

```diff
@@ -360,6 +360,48 @@ fun calculateJettonWalletAddress(
 }
 ```
 
+### Onchain metadata creation
+
+```tact
+/// https://github.com/ton-blockchain/TEPs/blob/master/text/0064-token-data-standard.md#jetton-metadata-example-offchain
+fun composeJettonMetadata(
+    name: String,        // full name
+    description: String, // text description of the Jetton
+    symbol: String,      // "stock ticker" symbol without the $ prefix, like USDT or SCALE
+    image: String,       // link to the image
+    // There could be other attributes, see:
+    // https://github.com/ton-blockchain/TEPs/blob/master/text/0064-token-data-standard.md#jetton-metadata-attributes
```

Comment:
I would also add this link https://github.com/getgems-io/nft-contracts/blob/main/docs/metadata.md

However onchain NFT attributes are not supported by getgems.